pkg/awsclient/client_test.go

```diff
@@ -0,0 +1,52 @@
+package awsclient
```

Comment:
Indentation in this file looks very strange - probably should throw `gofmt` at it to get it consistent.

Reply:
Consider it done :)